bot/src/infrastructure/repositories/KillRepositoryFactory.ts

```
@@ -0,0 +1,194 @@
+    
+    const current: KillRepositoryImplementation = 
+      envImplementation || (envOptimized ? 'optimized' : recommended);
+
+    return {
+      currentImplementation: current,
+      recommendedImplementation: recommended,
+      environmentOverride: envImplementation || (envOptimized ? 'optimized' : undefined),
+      productionReady: current === 'optimized',
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)